test/integration/full/landmark-main-is-top-level/frames/level2-a.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html id="violation4">
+<head>
+	<meta charset="utf8">
+	<script src="/axe.js"></script>
+</head>
+<body>
+	<p>This iframe is also a violation</p>
+	<div role = "form">
```

Comment:
This shouldn't be a failure. Neither should role="application". Make sure there is a test for that too.